### ISSUE TITLE
Create your own library (without training new models) example does not work.

### ISSUE THREAD

**Issue by mapio** [iomega/ms2query#211]
Running the "Create your own library (without training new models)" halts with an error at 

```
cleaned_library_spectra = clean_normalize_and_split_annotated_spectra(library_spectra, ion_mode_to_keep="")[0]
```

probably due to the fact that `ion_mode_to_keep` is only allowed `positive` or `negative` values due to an `assert` in the code.

**Comment by mapio** [comment]
Moreover, `LibraryFilesCreator` seems to expect some of the files to be in the output directory. The call in the example does not work (leads to a file not found error). It seems that

```
library_creator = LibraryFilesCreator(cleaned_library_spectra,
                                      output_directory=directory_for_library_and_models, 
                                      ms2ds_model_file_name=os.path.join(directory_for_library_and_models, ms2ds_model_file_name), 
                                      s2v_model_file_name=os.path.join(directory_for_library_and_models, s2v_model_file_name), ) 
```

fixes it. Not sure it is the right place to put the fix, other functions seems to prepend the output path themselves.

**Comment by niekdejonge** [comment]
Regarding your first comment. We do actually expect ion_mode_to_keep to be "positive" or "negative". This is the case since our models are made to match against positive or negative mode spectra. Having spectra in the library that do not match with what the library expects, will give unreliable results.

**Comment by niekdejonge** [comment]
Regarding your second point. That is indeed incorrect. I will update the readme as you suggested and have a look if we can make this a bit more robust/intuitive.

Sorry for the inconvenience. We thoroughly test our main code, but we are not yet able to automatically test our example scripts. Therefore some updates to the code are not reflected in the readme. I will also have a look at adding automatic checks for this as well, since it is crucial for new users that this reflects the current version of the code.

**Comment by niekdejonge** [comment]
Ah, now I also understand your first remark. We did not specify ion_mode_to_keep=ionisation_mode. Thanks for noting this!

**Comment by niekdejonge** [comment]
Fixed with #216
